README.md

```diff
@@ -150,37 +152,34 @@ runner:
 
 The test runner should produce a `test_report.json`
 file containing the results of the test executions. The structure of the test
-report should mirror that of the original test cases, with an additional
-attribute `result` added to each test object. This attribute should contain the
-set of rows returned by the implementation when evaluating the test. Ensure
-the result accurately reflects the output of your implementation for each
-test, facilitating a straightforward comparison between expected and actual
-outcomes.
+report is a map where:
+- each key is the name of a test file,
+- each value is a map with a single `tests` list,
+- each element of the `tests` list has a `name` and a `result` fields, reporting
+  whether the `name` test `passed` or not. It can optionally have a
+  `failureReason` text field as well.
```

Comment:
This should be updated to reflect the current expectations of the test report, as per the comment here: https://github.com/FHIR/sql-on-fhir-v2/pull/235#pullrequestreview-2030237672

Reply:
Thanks for pointing this out. I'll update this once I update #235 (and test it as you suggested).

Reply:
Done.